plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BaseBigQueryConnectorTest.java

```diff
@@ -265,7 +265,6 @@ protected Optional<DataMappingTestSetup> filterDataMappingSmokeTestData(DataMapp
             case "timestamp":
             case "timestamp(3)":
             case "timestamp(3) with time zone":
-            case "timestamp(6) with time zone":
```

Comment:
Ok go it so the tests fail because of this. Tracing the tests I do see that tests use invalid TIMESTAMP literals for BigQuery

https://github.com/trinodb/trino/blob/158f0b2f8748edf632a88f879d01ae208f08ed7d/testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java#L5169

Not really sure what do to here, add back this, or override @ebyhr?

Reply:
It means you need to fix this PR, (NOT test).

Reply:
Help me understand then a bit, there are no docs. isnt that literal `TIMESTAMP '2020-02-12 15:03:00 +01:00'` used in an BigQuery query?

Reply:
> It means you need to fix this PR, (NOT test)

TBF this is not really what I meant.

Reply:
addressed.